src/app/build.gradle.kts

```diff
@@ -20,6 +20,9 @@ dependencies {
     implementation("net.logstash.logback:logstash-logback-encoder:6.6")
     testImplementation("org.springframework.boot:spring-boot-starter-test")
     testImplementation("org.mockito.kotlin:mockito-kotlin:3.1.0")
+    testImplementation("org.leadpony.justify:justify:1.1.0")
+    testImplementation("org.leadpony.joy:joy:1.1.0")
+    testImplementation("com.fasterxml.jackson.module:jackson-module-kotlin")
```

Comment:
Maybe worth pinning this? Everything else is pinned (excluding spring and kotlin itself, where versions are inherited from `kotlinVersion` val).

Reply:
good idea